HISTORY.rst

```diff
@@ -6,7 +6,7 @@ History
 Pending release
 ---------------
 
-.. Insert new release notes below this line
```

Comment:
leave this comment

Reply:
It causes an issue with `multilint` : `warning: check: Explicit markup ends without a blank line; unexpected unindent. (line 226)`

Reply:
You need a blank line after comments in reStructuredText

Reply:
It wasn't obvious, thanks for the explanation :)